### DISCUSSION TITLE
[BUG] [SP-API] [GET_FBA_FULFILLMENT_INVENTORY_HEALTH_DATA always return FATAL]

### DISCUSSION THREAD

**Discussion by freedom64life** [amzn/selling-partner-api-models#3079] [General]
Starting from 2022-10-21 ，i get sp-api report type(GET_FBA_FULFILLMENT_INVENTORY_HEALTH_DATA ) always return  fatal ，is the same as the other programmer? 

the same problem report type GET_EXCESS_INVENTORY_DATA，GET_FBA_INVENTORY_AGED_DATA return fatal.

i think the problem is related to this [page](https://sellercentral-europe.amazon.com/help/hub/reference/external/200537120?ref=efph_200537120_cont_200885390&locale=en-GB)

but now the report is stop ,there is an other report replace the old one?

**Comment by datitran** [comment]
We also have this issue what's the new report type for this one?

**Comment by yutveg** [comment]
Seeing the same issue here! Looks like the reports started failing last Friday

**Comment by sungolivia** [comment]
Hey @yutveg @datitran @l172942736 any other reports seeing this issue aside from the ones mentioned in this thread already?

**Comment by yutveg** [comment]
Not any additional from me, I'm only seeing the GET_EXCESS_INVENTORY_DATA，GET_FBA_INVENTORY_AGED_DATA failing so far

**Comment by datitran** [comment]
Same here only GET_FBA_FULFILLMENT_INVENTORY_HEALTH_DATA is failing for us. I already filed a ticket on Seller Central as well but they said everything is okay 😅 So I was happy that we are not the only ones with that problem as everything is also green on status.mws.amazon.com.

**Comment by kelerensheng** [comment]
Amazon previously announced that this report will be replaced by a new report "GET_FBA_INVENTORY_PLANNING_DATA" [see May 2022 SP-API Release Announcement](https://github.com/amzn/selling-partner-api-docs/discussions/2289)
![image](https://user-images.githubusercontent.com/37256102/197672659-d3b7bf34-63b7-4c28-825b-7e80842cb9f7.png)

**Comment by datitran** [comment]
Okay but weirdly I'm not able to access this report even though I have all the permission I get this here: `[{'code': 'Unauthorized', 'message': 'Access to the resource is forbidden', 'details': ''}]`

**Comment by freedom64life** [comment]
> Amazon previously announced that this report will be replaced by a new report "GET_FBA_INVENTORY_PLANNING_DATA" [see May 2022 SP-API Release Announcement](https://github.com/amzn/selling-partner-api-docs/discussions/2289) ![image](https://user-images.githubusercontent.com/37256102/197672659-d3b7bf34-63b7-4c28-825b-7e80842cb9f7.png)

that is it,thank you very much~
